Dockerfile

```diff
@@ -26,6 +66,9 @@ RUN adduser -u 9000 -D -h /usr/src/app -s /bin/false app
 COPY . /usr/src/app
 RUN chown -R app:app /usr/src/app
 
+# ENV PURE_HASH=1
+# ENV CODECLIMATE_PROFILE=1
```

Comment:
Sure... I'd like to leave the PURE_HASH as a TODO line... it does come out ahead in my measurements.